src/ukify/ukify.py

```diff
@@ -918,31 +963,46 @@ def make_uki(opts):
             # Merge the .sbat sections from stub, kernel and parameter, so that revocation can be done on either.
             input_pes = [opts.stub, linux]
             if not opts.sbat:
-                opts.sbat = ["""sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+                opts.sbat = [
+                    """sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
 uki,1,UKI,uki,1,https://uapi-group.org/specifications/specs/unified_kernel_image/
-"""]
+"""
+                ]
```

Comment:
`# fmt: skip` ?

Reply:
I went for a slightly different approach and factored out the unwieldy strings into a constant just above the function and added a line continuation to make it more readable, PTAL.